WalletWasabi.Gui/Behaviors/GreyOutOnDisabledBehavior.cs

```diff
@@ -19,18 +19,19 @@ protected override void OnAttached()
 
 			Disposables = new CompositeDisposable
 			{
-				AssociatedObject.GetObservable(InputElement.IsEnabledProperty).Subscribe(enabled =>
-				{
-					if (enabled)
+				AssociatedObject.GetObservable(InputElement.IsEnabledProperty)
```

Comment:
`.GetObservable(InputElement.IsEnabledProperty)`
in the next line

Reply:
Done.